src/components/views/messages/MessageTimestamp.tsx

```diff
@@ -48,6 +48,7 @@ export default class MessageTimestamp extends React.Component<IProps> {
                 className="mx_MessageTimestamp"
                 title={formatFullDate(date, this.props.showTwelveHour)}
                 aria-hidden={true}
+                min-height="1rem"
```

Comment:
There are no `min-height` attributes on a `span` HTML element.

My hunch is that we would have to update the `mx_EventTile_line` CSS class rather than the message timestamp. It's positioned absolutely and therefore will not affect the height of the event tile